tests/pipeline/test_downsampling.py

```diff
@@ -726,6 +730,8 @@ class TestDownsampledRowwiseOperation(WithAssetFinder, ZiplineTestCase):
 
     dates = pd.date_range(START_DATE, END_DATE)
 
+    ASSET_FINDER_COUNTRY_CODE = '??'
```

Comment:
I kept this as `'??'` because this suite actually just uses `date_range` as its calendar. I don't want to use US for this since that would signal to me that I should expect the US calendar. Do you have thoughts on a better code to use for "just testing"/"synthetic"?